### ISSUE TITLE
Goldwarden spawns python processes until pc crash or processes are manually killed 

### ISSUE THREAD

**Issue by ghandigarcia** [quexten/goldwarden#218]
Hey, big fan of this project since I came across it a few months ago! 

I have a bit of an issue that I think is related to Goldwarden, though I'm not entirely sure. I also have messed something up somewhere, that's always a possibility :)

It is the only application that I have installed with python code that I've found to cause the following behavior; 
with Goldwarden auto-starting on boot, or when starting it via either gui or CLI, a number of python processes roughly ~8mb in size starts being spawned. They keep spawning until the laptop either crashes or I manually kill the processes. 
[Screencast from 15-05-24 14:54:10.webm](https://github.com/quexten/goldwarden/assets/66558260/b76032c8-6a2e-4224-8c15-1f9d5db6527a)

Screenshot of what it looks like when I run `flatpak run com.quexten.Goldwarden`
![Screenshot from 2024-05-15 15-25-42](https://github.com/quexten/goldwarden/assets/66558260/819973ae-5fd1-4561-afcd-bf160ca2b647)

**Comment by hulloanson** [comment]
Same experience here. There are upwards of 100 `status.py` processes when I check and killing them gave me back 14GiB of memory.

**Comment by quexten** [comment]
What desktop environment are you on?

**Comment by quexten** [comment]
#223 should fix this. I think your de does not implement the background portal (or just v1), and so the status setting dbus call fails, and the process never exits, just idling. Since status.py is spawned quite frequently (and expected to exit immediately), this causes the mess you are seeing.

**Comment by ghandigarcia** [comment]
> What desktop environment are you on?

I'm on Gnome, version 42.9. Let me know if there's anything else you need or if you'd like me to test something. I'll do my best to help!

**Comment by hulloanson** [comment]
> #223 should fix this. I think your de does not implement the background portal (or just v1), and so the status setting dbus call fails, and the process never exits, just idling. Since status.py is spawned quite frequently (and expected to exit immediately), this causes the mess you are seeing.


Did find these in logs when running goldwarden gui in cli with `flatpak run`.

```plaintext
g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such interface “org.freedesktop.portal.Background” on object at path /org/freedesktop/portal/desktop (19)
```

I am using LXDM with openbox.

Copying the fix in #223 did fix the issue. Kudos to the quick resolution!

**Comment by ghandigarcia** [comment]
Just tested the fix in [#223](https://github.com/quexten/goldwarden/pull/223) as well, it looks all good now. Thanks!
